pallets/dkg/src/functions.rs

```diff
@@ -69,8 +69,8 @@ impl<T: Config> Pallet<T> {
 	/// an error.
 	pub fn verify(data: JobResult) -> DispatchResult {
 		match data {
-			JobResult::DKG(info) => Self::verify_generated_dkg_key(info),
-			JobResult::DKGSignature(info) => Self::verify_dkg_signature(info),
+			JobResult::DKGPhaseOne(info) => Self::verify_generated_dkg_key(info),
```

Comment:
Should really be TSSPhaseOne, TSSPhaseTwo. We will eventually have a threshold encryption DKG so DKG isn't the right term here to be used entirely.

Reply:
We could do DkgTssPhaseOne, DkgTssPhaseTwo and prep for DkgTd... (for threshold-decryption)